src/components/NcRichContenteditable/NcAutoCompleteResult.vue

```diff
@@ -67,6 +67,10 @@ export default {
 			type: String,
 			required: true,
 		},
+		iconUrl: {
```

Comment:
Since this is the avatar url, I would also call it `avatarUrl`.

Reply:
Then it conflicts with the computed and the CSS avatar option is also called `icon` so I tried to keep it simple and inline

Reply:
We could come up with something like `predefinedUrl`, as we provide here a ready-to-download URL